src/test/java/com/wcc/platform/factories/SetupFactories.java

```diff
@@ -33,6 +35,9 @@
 /** Setup Factory tests. */
 public class SetupFactories {
 
+  public static final int DEFAULT_CURRENT_PAGE = 1;
```

Comment:
I don't think you are using these constants.

Reply:
I am using them in the controller